util/chplenv/chpl_home_utils.py

```diff
@@ -23,13 +23,12 @@ def get_chpl_third_party():
     return chpl_third_party
 
 # Get the chpl-venv install directory:
-# $CHPL_HOME/third-party/chpl-venv/install/$bin_subdir/$py_version/chpl-virtualenv
+# $CHPL_HOME/third-party/chpl-venv/install/$bin_subdir/chpl-virtualenv
```

Comment:
We likely don't even need the bin_subdir anymore since we're just getting pure python code, but I don't mind leaving that for now.